README.md

```diff
@@ -230,6 +230,7 @@ export ASB_SPOKE_VNET_ID=$(az deployment group show -g $ASB_RG_SPOKE -n spoke-$A
 
 ### this section takes 15-20 minutes
 
+# Note: If using an existing Log Analytics workspace, pass in the laWorkspaceName and laResourceGroup paramaters to the following command
```

Comment:
Can we set variables for them to empty and go ahead and create the next command with the assignment using the vars?  Then we can tell folks to set the vars with the correct values?  If that won't work then I would suggest two different blocks one with the params and one without and have them labeled appropriately.

Reply:
Done